### DISCUSSION TITLE
dep-check: Show warning for react-native 0.70x when @types/react-native was found as dependency

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/rnx-kit#1898] [Ideas]
`@types/react-native` was merged into react-native repo 

> The current plan is to release `@types/react-native` for `0.70` and `0.71` while also maintaining types here. This will result in some double maintenance until `0.72` but will give community time to move off of `@types/react-native`.

See: https://github.com/facebook/react-native/pull/34614

So it's not need any more to install `@types/react-native` for new versions on react native

**Comment by tido64** [comment]
Sounds like a good idea. I've created an issue for it: https://github.com/microsoft/rnx-kit/issues/1899
